config/registry_image_pruner/image_pruner/prune_images.py

```diff
@@ -61,34 +79,33 @@ def delete_image_tag(quay_token: str, namespace: str, name: str, tag: str) -> No
             raise(ex)
 
 
-def remove_tags(tags: Dict[str, Any], quay_token: str, namespace: str, name: str, dry_run: bool = False) -> None:
-    image_digests = [image["manifest_digest"] for image in tags.values()]
+def remove_tags(tags: List[Dict[str, Any]], quay_token: str, namespace: str, name: str, dry_run: bool = False) -> None:
+    image_digests = [image["manifest_digest"] for image in tags]
     tag_regex = re.compile(r"^sha256-([0-9a-f]+)(\.sbom|\.att|\.src|\.sig)$")
     for tag in tags:
         # attestation or sbom image
-        if (match := tag_regex.match(tag)) is not None:
+        if (match := tag_regex.match(tag["name"])) is not None:
             if f"sha256:{match.group(1)}" not in image_digests:
                 if dry_run:
-                    LOGGER.info("Image %s from %s/%s should be removed", tag, namespace, name)
+                    LOGGER.info("Image %s from %s/%s should be removed", tag["name"], namespace, name)
                 else:
-                    LOGGER.info("Removing image %s from %s/%s", tag, namespace, name)
-                    delete_image_tag(quay_token, namespace, name, tag)
+                    LOGGER.info("Removing image %s from %s/%s", tag["name"], namespace, name)
```

Comment:
Is it really image or tag?

Reply:
well :D actually we are removing here those additional artifacts

but we are printing the whole artifact name, so it is clear